install-check.sh

```diff
@@ -2,20 +2,38 @@
 HOST=$1
 VERSION_FILE=$2
 
-if [[ -z `grep "__version__" version_check_$HOST.txt` ]]
-then 
-	echo "배포 대상 edge device의 version 파일을 읽어올 수 없습니다."
+extract_version() {
+	local check_file=$1
+	
+	raw_version_data=`grep $version_key_string $check_file`
+	if [[ -z $raw_version_data ]]
+	then
+		echo "배포 대상 edge device의 version 파일을 읽어올 수 없습니다."
+		exit 1
+	fi
+	version=`echo $raw_version_data | sed 's/[^a0-9.]//g'`
+	echo $version
+}
+
+if [[ $VERSION_FILE == "package.json" ]]
+then
+	version_key_string="version"
```

Comment:
```suggestion
	version_key_string="\"version\""
```
package.json 파일은 버전 외에도 설명이나 이런저런 데이터가 많아 version 텍스트로만 검색하면 abc-version, versioning 등 다른 단어나 잘못된 라인을 가져오는 문제가 생길것 같습니다.. JSON 포멧이기 때문에 " " 문자를 포함해서 정확히  "version" 스트링만을 검색하면 어떨까요?

Reply:
오호! 넵 https://github.com/teamdable/vv-deploy-actions/pull/13/commits/6f008287624c045e89854ef04ac960a3c75eeec1에서 반영했습니다 ~!